backend/workflow_manager/endpoint/models.py

```diff
@@ -32,7 +33,7 @@ class ConnectionType(models.TextChoices):
     connection_type = models.CharField(
         choices=ConnectionType.choices,
         blank=True,
-        db_comment="Connection type (Filesystem, Database or API)",
+        db_comment="Connection type (Filesystem, Database or API or Queue)",
```

Comment:
@vishnuszipstack NIT: There is an unwated `or` before API in the db_comment.

Modified comment: `Connection type (Filesystem, Database, API or Queue)`

Reply:
done @tahierhussain